{
	"id": "CVE-2024-25631",
	"sourceIdentifier": "security-advisories@github.com",
	"vulnStatus": "Awaiting Analysis",
	"published": "2024-02-20T18:15:53.117",
	"lastModified": "2024-02-20T19:50:53.960",
	"descriptions": [
		{
			"lang": "en",
			"value": "Cilium is a networking, observability, and security solution with an eBPF-based dataplane. For Cilium users who have enabled an external kvstore and Wireguard transparent encryption, traffic between pods in the affected cluster is not encrypted. This issue affects Cilium v1.14 before v1.14.7 and has been patched in Cilium v1.14.7. There is no workaround to this issue."
		}
	],
	"references": [
		{
			"source": "security-advisories@github.com",
			"url": "https://docs.cilium.io/en/stable/installation/k8s-install-external-etcd/#when-do-i-need-to-use-a-kvstore"
		},
		{
			"source": "security-advisories@github.com",
			"url": "https://docs.cilium.io/en/stable/security/network/encryption-wireguard/#encryption-wg"
		},
		{
			"source": "security-advisories@github.com",
			"url": "https://github.com/cilium/cilium/releases/tag/v1.14.7"
		},
		{
			"source": "security-advisories@github.com",
			"url": "https://github.com/cilium/cilium/security/advisories/GHSA-x989-52fc-4vr4"
		}
	],
	"metrics": {
		"cvssMetricV31": [
			{
				"source": "security-advisories@github.com",
				"type": "Secondary",
				"cvssData": {
					"version": "3.1",
					"vectorString": "CVSS:3.1/AV:A/AC:H/PR:N/UI:N/S:C/C:H/I:N/A:N",
					"attackVector": "ADJACENT_NETWORK",
					"attackComplexity": "HIGH",
					"privilegesRequired": "NONE",
					"userInteraction": "NONE",
					"scope": "CHANGED",
					"confidentialityImpact": "HIGH",
					"integrityImpact": "NONE",
					"availabilityImpact": "NONE",
					"baseScore": 6.1,
					"baseSeverity": "MEDIUM"
				},
				"exploitabilityScore": 1.6,
				"impactScore": 4
			}
		]
	},
	"weaknesses": [
		{
			"source": "security-advisories@github.com",
			"type": "Secondary",
			"description": [
				{
					"lang": "en",
					"value": "CWE-311"
				}
			]
		}
	]
}
